src/style.css

```diff
@@ -0,0 +1,90 @@
+body {
+  margin: 0;
+}
+
+.container {
+  margin: 0;
+  display: flex;
+  width: 96px;
+  height: 96px;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  align-self: auto;
+
+}
+
+.stars__star {
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(-n+5) {
```

Comment:
For the stars--1 class, remove the nth-child(-n+5) selector as it is unnecessary.